packages/terra-alert/src/Alert.jsx

```diff
@@ -0,0 +1,280 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
```

Comment:
use `classnames/bind`. It allows you to skip the `styles.x` part and just use `x`. See https://github.com/JedWatson/classnames#alternate-bind-version-for-css-modules